src/components/utils/OffchainUtils.ts

```diff
@@ -6,6 +6,7 @@ const createUrl = (varName: string, defaultUrl: string = 'http://localhost:3001'
 
 export const offchainUrl = createUrl('OFFCHAIN_URL');
 export const ipfsUrl = createUrl('IPFS_URL', getEnv('OFFCHAIN_URL'));
+export const offchainWs = getEnv('OFFCHAIN_WS') as string
```

Comment:
This `getEnv()` should return a string already, why do we need a cast to `as string`?
```suggestion
export const offchainWs = getEnv('OFFCHAIN_WS') as string
```